airbyte-server/src/main/java/io/airbyte/server/errors/UnsupportedProtocolVersion.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.server.errors;
+
+import io.airbyte.commons.version.Version;
+
+public class UnsupportedProtocolVersion extends KnownException {
```

Comment:
Nit:  Should this have `Exception` in the class name to make it clear that it represents an exception?

Reply:
done